iothub/service/src/DigitalTwin/DigitalTwinClient.cs

```diff
@@ -10,6 +10,10 @@
 using Microsoft.Azure.Devices.Generated;
 using Microsoft.Rest;
 using Newtonsoft.Json;
+using Microsoft.Azure.Devices.DigitalTwin.Authentication;
+using Azure;
+using Azure.Core;
+using PnpDigitalTwin = Microsoft.Azure.Devices.Generated.DigitalTwin;
```

Comment:
This is because of the namespace conflict with Azure.Core DigitalTwin :D

Reply:
Why would we have a namespace conflict with our track2 ADT SDK?

Reply:
Because the AzureSasToken is in the "Azure" namespace, which will include everything under that. I don't see why else it complained about a conflict with a namespace.

Reply:
If we don't have a reference to Azure.DigitalTwins.Core, I don't see how we could have a type conflict.